examples/scripts/my_test_facts.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python
+
+# Copyright: (c) 2020, Your Name <YourName@example.org>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+---
+module: my_test_facts
+
+short_description: This is my test facts module
+
+version_added: "2.11"
+
+description: This is my longer description explaining my test info module.
+
+author:
+    - Your Name (@yourGitHubHandle)
+'''
+
+EXAMPLES = r'''
+- name: Return ansible_facts
+  my_test_facts:
+'''
+
+RETURN = r'''
+ansible_facts:
+  description: facts to add to ansible_facts
+  returned: always
+  type: complex
+  contains:
+    foo:
+      description: foo facts about operating system
```

Comment:
```suggestion
      description: Foo facts about operating system
```
etc.
and would be cool to see `sample` subsections here